Lessen_5_hw/task_1.js

```diff
@@ -0,0 +1,20 @@
+/*
+  Пример в консоли:
+  :)
+  :):)
+  :):):)
+  :):):):)
+  :):):):):)
+Рекоммендация: попробуйте метод .repeat()
+*/
+//Создайте цикл, который выведет в консоль цифры от 10 до 0
+for (let i = 0; i <= 10; i += 1) {
+	console.log(i)
+}
+console.log()
```

Comment:
Follow task description - `from 10 to 0`
```suggestion
for (let i = 0; i <= 10; i += 1) {
	console.log(i)
}
```